crypto/evp/e_aes.c

```diff
@@ -611,22 +611,22 @@ void aes_t4_decrypt(const unsigned char *in, unsigned char *out,
  */
 void aes128_t4_cbc_encrypt(const unsigned char *in, unsigned char *out,
                            size_t len, const AES_KEY *key,
-                           unsigned char *ivec);
+                           unsigned char *ivec, int /*unused*/);
```

Comment:
Just a note: C allows extra trailing arguments without screwing up the stack frames.  That's how the `...` varargs is supported.  With proper prototypes, this is allowed to change but rarely does in my experience.

On SPARC, arguments are passed via registers so it's less of a concern again.

Reply:
Great, thanks!

When I saw the original commit in master fixing this I was a bit surprised because I was expecting it to cause problems in stack frames.

Your comment is very useful!